Containerfile

```diff
@@ -1,5 +1,5 @@
 FROM registry.fedoraproject.org/fedora:39 AS builder
-RUN dnf install -y git-core golang gpgme-devel libassuan-devel && mkdir -p /build/bib
+RUN dnf install -y git-core golang gpgme-devel libassuan-devel btrfs-progs-devel lvm2-devel && mkdir -p /build/bib
```

Comment:
We should be just build-taging these out.  You don't need to support btrfs or lvm.

Reply:
Thank you! Colin fixed this already, once I rebased to "main" this will go away.